SECURITY.md

```diff
@@ -15,7 +15,7 @@ If you contact us regarding a serious issue:
 
 The best way to report an issue is by contacting us via email at info@balazsorban.com or me@iaincollins.com and yo@ndo.dev, or raise a public issue requesting someone get in touch with you via whatever means you prefer for more details. (Please do not disclose sensitive details publicly at this stage.)
 
-> For less serious issues (e.g. RFC compliance for unsupported flows or potential issues that may cause a problem in the future) it is appropriate to submit these these publically as bug reports or feature requests or to raise a question to open a discussion around them.
+> For less serious issues (e.g. RFC compliance for unsupported flows or potential issues that may cause a problem in the future) it is appropriate to submit these publicly as bug reports or feature requests or to raise a question to open a discussion around them.
```

Comment:
That was spelled correctly initially. `Publically` :+1:

Reply:
@ndom91 I could update if needed. The original spelling is `publicly` according to https://capitalizemytitle.com/publicly-vs-publically/ and also
<img width="688" alt="image" src="https://user-images.githubusercontent.com/43613/180333520-29c5a872-c3ab-4994-ba73-e492565bff23.png">

Reply:
@ndom91 are we good here?